third_party/terraform/resources/resource_dataproc_cluster.go.erb

```diff
@@ -508,6 +511,42 @@ by Dataproc`,
 								},
 							},
 						},
+<% unless version == 'ga' -%>
+						"lifecycle_config": {
+							Type:             schema.TypeList,
+							Optional:         true,
+							MaxItems:         1,
+							AtLeastOneOf:     clusterConfigKeys,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"idle_delete_ttl": {
+										Type:     schema.TypeString,
+										Optional: true,
+										AtLeastOneOf: []string{
+											"cluster_config.0.lifecycle_config.0.idle_delete_ttl",
+											"cluster_config.0.lifecycle_config.0.auto_delete_time",
+										},
+									},
+									"idle_start_time": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+									// the API also has the auto_delete_ttl option in its request, however,
+									// the value is not returned in the response, rather the auto_delete_time
```

Comment:
I think this is now returned in the response when it's passed in the request. The bug I filed didn't get fixed but maybe they snuck a fix in there. Can you double check?

Reply:
I double-checked, and per our slack conversation, it is still not available.  However, I've added the tests (commented out) for when it does get added.  I can remove them if you want, but it includes a few things I wanted to test, so that when it _does_ get added in, we keep the logic there (updating from `ttl` to `time` to new `time` to `ttl` to new `ttl`) just so it's thoroughly tested.  Not sure if that's overkill?

Reply:
Confirmed that `auto_delete_time` is returned and `auto_delete_ttl` is not.